framework/TApplication.php

```diff
@@ -1023,6 +1024,7 @@ public function applyConfiguration($config, $withinService = false)
 	 * Loads configuration and initializes application.
 	 * Configuration file will be read and parsed (if a valid cached version exists,
 	 * it will be used instead). Then, modules are created and initialized;
+	 * Finalize any modules by calling onInitComplete, mainly for CLI initialization.
```

Comment:
Please reword this; onInitComplete is an event, so the use doesn't "call" it.
Something like "module initialization can be finalized in the onInitComplete event, this is particularly useful for CLI applications"